ExampleApp/src/main/AndroidManifest.xml

```diff
@@ -25,6 +25,10 @@
                 android:name="AA_DB_VERSION"
                 android:value="1"/>
 
+        <service
+                android:name=".MyIntentService"
```

Comment:
Thanks and in meantime I think I can have some more gingerbread testing
time in next 24 hrs.... So I will update you (and if needed the code) about
my findings.
On 16-Feb-2015 12:59 pm, "Prasham Trivedi" <prash2488@gmail.com> wrote:

> Oops forgot to cleanup... Looks like new pull request has to be submitter
> On 16-Feb-2015 12:55 pm, "Dino Kovač" <notifications@github.com> wrote:
>
>> In ExampleApp/src/main/AndroidManifest.xml
>> <https://github.com/infinum/android_dbinspector/pull/9#discussion_r24734743>
>> :
>>
>> > @@ -25,6 +25,10 @@
>> >                  android:name="AA_DB_VERSION"
>> >                  android:value="1"/>
>> >
>> > +        <service
>> > +                android:name=".MyIntentService"
>>
>> This doesn't seem to match the name of the service. Am I missing
>> something?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/infinum/android_dbinspector/pull/9/files#r24734743>.
>>
>